pkg/api/options.go

```diff
@@ -9,4 +9,5 @@ type Options struct {
 	DBConnectionString  string `long:"db-connection-string" env:"DB_CONNECTION_STRING"`
 	AdminAPIKey         string `long:"admin-api-key" env:"ADMIN_API_KEY" description:"admin api key"`
 	Servicedomain       string `long:"service-domain" env:"SERVICE_DOMAIN" description:"the domain at which the service is served" default:"third-rail.services.smartatransit.com"`
+	RailRunner          bool   `long:"rail-runner" env:"RAIL_RUNNER" description:"enable the rail runner"`
```

Comment:
I added an option to disable the rail-runner, because it produces a lot of logs if it doesn't have the right configuration (which, for right now, it doesn't)